docs/docs/tutorials/get-started/customize-sign-in-experience.md

```diff
@@ -3,3 +3,49 @@ sidebar_position: 3
 ---
 
 # Customize sign-in experience
+
+In the "Get Started" tab, click the "Create" button on the right, and the browser will redirect to the Sign-in Experience tab.
+
+## Sign-in Preview
+
+:::tip Do you know...
+You can preview the result in real-time! No matter what config you update, the right-side UI will change accordingly in a moment.
```

Comment:
nit: `No matter what you update on the left, the right side UI will immediately update to synchronize the change.`
Just for your reference

Reply:
you are English master

Reply:
updated